spec/indexers/descriptive_metadata/pub_year_1e_spec.rb

```diff
@@ -307,7 +309,8 @@
         }
       end
 
-      it 'selects first date' do
+      # Test skipped until stanford-mods indexes dateOther
```

Comment:
it may never do that;  I would reword "test skipped because stanford-mods doesn't index dateOther"